doc/user-guide/src/installation/other.md

```diff
@@ -3,9 +3,10 @@
 The primary installation method, as described at <https://rustup.rs>, differs
 by platform:
 
-* On Windows, download and run the [`rustup-init.exe` built for the
-  `x86_64-pc-windows-msvc` target][setup]. In general, this is the build of
-  `rustup` one should install on Windows. This will require the Visual C++
+* On Windows, download and run the `rustup-init.exe` built for the
+  [`x86_64-pc-windows-msvc`] or [`aarch64-pc-windows-msvc`] target.
+  In general, this is the build of `rustup` one should install on Windows,
+  depending on the OS architecture. This will require the Visual C++
```

Comment:
```suggestion
* On Windows, download and run the `rustup-init.exe` built for the
  [`x86_64-pc-windows-msvc`] or [`aarch64-pc-windows-msvc`] target,
  depending on your OS architecture. In general, this is the build of
  `rustup` one should install on Windows. This will require the Visual C++
```

Reply:
Done!